lib/dockerregistry/inventory.go

```diff
@@ -1276,6 +1277,11 @@ func (sc *SyncContext) ReadRegistries(
 					fmt.Printf("digest %s: %s\n", digest, err)
 				}
 				sc.DigestMediaType[Digest(digest)] = mediaType
+
+				// Store ImageSize
+				if sc.DigestImageSize != nil {
```

Comment:
Because some tests make a SyncContext without the DigestImageSize field which causes the line "sc.DigestImageSize[Digest(digest)] = int(mfestInfo.Size)" to fail